.github/workflows/testAndPublish.yml

```
@@ -163,17 +157,17 @@ jobs:
     if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/beta' }}
```

Wouldn't it be more efficient to check if the Crowdin ID is defined up here, rather than at each step?
We needed to check at each step because the `env` context isn't accessible from `jobs.<job_id>.if`, but `vars` is available there. https://docs.github.com/en/actions/reference/accessing-contextual-information-about-workflow-runs#context-availability
```suggestion
    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/beta' && vars.CROWDIN_PROJECT_ID }}
```

---

nice catch